static/api-spec.yaml

```
@@ -0,0 +1,754 @@
+                                                    { 'type': 'string' },
+                                                },
+                                              'additionalProperties':
+                                                { 'type': 'string' },
+                                            },
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+              'tags': ['System'],
+            },
+        },
+      '/dv/{configHash}':
+        {
+          'get':
+            {
+              'operationId': 'DVController_getClusterDefinition',
+              'summary': 'Retrieve a Distributed Validator Cluster proposal',
+              'description': "This endpoint allows a charon client or launchpad interface to retrieve the terms of a proposed DKG. Once all operators listed in the DKG have submitted signed approvals to the terms, this object will be ready for a cluster of operators to use as part of a DKG ceremony. If the objects in the `operators` array are not fully populated, these operators need to use the [PUT request](#/Distributed-Validators/operation/DVController_updateClusterDefinition) to upload their charon client's public key and a signature from their address to indicate their acceptance of the terms.",
+              'parameters':
+                [
+                  {
+                    'name': 'configHash',
+                    'required': true,
+                    'in': 'path',
+                    'description': 'The `config_hash` calculated for a cluster definition.',
+                    'schema': { 'type': 'string' },
+                  },
+                ],
+              'responses':
+                {
+                  '200':
+                    {
+                      'description': 'The cluster object',
+                      'content':
+                        {
+                          'application/json':
+                            {
+                              'schema':
+                                {
+                                  '$ref': '#/components/schemas/GETClusterDefinitionResponse',
+                                },
+                            },
+                        },
+                    },
+                },
+              'tags': ['Distributed Validators'],
+            },
+          'put':
+            {
+              'operationId': 'DVController_updateClusterDefinition',
+              'summary': 'Accept a proposed Distributed Validator Cluster',
+              'description': 'This endpoint is used by the operators present in the `operators` array of a cluster definition. These entites must submit a public key (in ENR form) to serve as their identity during the DKG, along with EIP712 signatures indicating their acceptance of the terms of this DKG.',
```

```suggestion
              'description': 'This endpoint is used by the operators present in the `operators` array of a cluster definition. These operators must submit a public key (in ENR form) to serve as their identity during the DKG, along with EIP712 signatures indicating their acceptance of the terms of this DKG.',
```

Confusing to now refer to them as entities